src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2641,7 +2662,7 @@ class CgroupUtils(object):
                             assigned[jobid]['memsw']['limit_in_bytes'] = \
                                 int(desc.readline())
                     else:
-                        pbs.logmsg(pbs.EVENT_DEBUG, '%s: No such file: %s' %
```

Comment:
Why reducing the log level here?

Reply:
I was finding during suspend/resume testing with pbs_cgroups hook, that if kernel has not configured memsw.limit_in_bytes, then we keep getting this "no such file memsw.limit_in_bytes" in the logs at the default level. This happens every time the get_assigned_cgroup_resources() is called, which is in a few places. I was seeing it on my local host And looking at the pbs_cgroups hook code, getting this message is not critical. So, it's best to relegate it to DEBUG4.